### PR TITLE
improved display pictures

### PR DESCRIPTION
**Pull request by modos189**

### CODE REVIEW

**Comment by j0k3r** [comment]
Screenshot?

File: src/Wallabag/CoreBundle/Resources/views/themes/material/public/css/main.css [hunk 1]
```diff
@@ -221,15 +221,11 @@ main ul.row {
     padding: 0px 0.75rem;
 }
 
-.data .card .card-content {
+.data .card .card-body {
     height: 22em;
     overflow: hidden;
 }
 
-img.preview {
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/public/css/main.css]: if you remove these lines, preview picture display is broken on article view (picture is too huge)

**Comment by nicosomb** [comment]
Before



After

File: src/Wallabag/CoreBundle/Resources/views/themes/material/public/css/main.css (hunk 1 above)
Comment (modos189) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/public/css/main.css]: To limit the size I use lines 265 to 275

**Comment by modos189** [comment]
Regard to the screenshot, I think, when you click on the picture better show the title and description, without the website address

**Comment by nicosomb** [comment]
No problem for that. 
Did you see my comment about article view?

**Comment by modos189** [comment]
About "img.preview"? Yes, i answered)

**Pull request merged**